dockerfiles/header_sync/startup_script.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+# Runs the migrations and start headerSync
+
+# Verify required args present
+MISSING_VAR_MESSAGE=" is required and no value was given"
+
+if test -z "$STARTING_BLOCK_NUMBER"
+then
+    echo STARTING_BLOCK_NUMBER $MISSING_VAR_MESSAGE
+    exit 1
+fi
+
+function testDatabaseVariables() {
+  for a in DATABASE_NAME DATABASE_HOSTNAME DATABASE_PORT DATABASE_USER DATABASE_PASSWORD
+  do
+    eval arg="$"$a
+    test $arg
+    if [ $? -ne 0 ]; then
+      echo $a $MISSING_VAR_MESSAGE
+      exit 1
+    fi
+  done
+}
+
+if test -z "$VDB_PG_CONNECT"; then
+  # Exits if the variable tests fail
+  testDatabaseVariables
+  if [ $? -ne 0 ]; then
+    exit 1
+  fi
+
+  # Construct the connection string for postgres
+  VDB_PG_CONNECT=postgresql://$DATABASE_USER:$DATABASE_PASSWORD@$DATABASE_HOSTNAME:$DATABASE_PORT/$DATABASE_NAME?sslmode=disable
+fi
+
+# Run the DB migrations
+echo "Connecting with: $VDB_PG_CONNECT"
+./goose -dir db/migrations postgres "$VDB_PG_CONNECT" up
+
+if [ $? -ne 0 ]; then
+  echo "Could not run migrations. Are the database details correct?"
+  exit 1
+fi
+
+echo "Starting headerSync..."
+# Fire up headerSync
+if [ $? -eq 0 ]; then
```

Comment:
What execution status is this clause checking? My bash is rusty, but `echo "Starting headerSync..."` would always return true anyway no? :woman_shrugging:

Reply:
Yeah just unnecessary copypasta there, removed